<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MulitCameraListItemWidget</class>
 <widget class="QWidget" name="MulitCameraListItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MulitCameraListItemWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="ClickableLabel" name="label_typeIcon">
     <property name="maximumSize">
      <size>
       <width>11</width>
       <height>12</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLabel" name="label_cameraName">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLabel" name="label_number">
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>14</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
